src/cookbook/editable-svg-icons.md

```diff
@@ -40,9 +40,9 @@ We'll create a base icon (`IconBase.vue`) component that uses a slot.
 </template>
 ```
 
-You can use this base icon as is- the only thing you might need to update is the `viewBox` depending on the `viewBox` of your icons. In the base, we're making the `width`, `height`, `iconColor`, and name of the icon props so that it can be dynamically updated with props. The name will be used for both the `<title>` content and its `id` for accessibility.
+Puede usar este ícono base tal como está, lo único que podría tener que cambiar es el `viewBox` dependiendo del `viewBox` de sus íconos. En la base, estamos creando como _props_ a `width`, `height`, `iconColor`, y al nombre del ícono, de tal manera que puedan ser actualizadas dinámicamente. El nombre será usado para el contenido de `<title>` y de su `id` para accesibilidad.
```

Comment:
I'm not a big of translating `making` to `creando` on this context, what do you think about this?
```suggestion
Puede usar este ícono base tal como está, lo único que podría tener que cambiar es el `viewBox` dependiendo del `viewBox` de sus íconos. En la base, estamos definiendo como _props_ a `width`, `height`, `iconColor`, y al nombre del ícono, de tal manera que puedan ser actualizadas dinámicamente. El nombre será usado para el contenido de `<title>` y de su `id` para accesibilidad.
```

Reply:
Agree!